distribution/bin/enso

```diff
@@ -1,5 +1,5 @@
 COMP_PATH=$(dirname "$0")/../component
```

Comment:
This file could be auto-generated so that we don't hardcode here the name of the module.